src/react/Router.jsx

```diff
@@ -70,8 +70,8 @@ export default ({ reactHistory: history, astarteClient, sessionManager, config,
           path="login"
           element={
             <Login
-              canSwitchLoginType={config.auth.length > 1}
-              defaultLoginType={config.default_auth || 'token'}
+              authOptions={config.auth}
+              defaultAuth={config.defaultAuth}
               defaultRealm={config.default_realm || ''}
```

Comment:
Reading a generic `config.auth` name, I would think it as a well defined configuration section: a single object to pass around such as
```typescript
{
  defaultRealm: 'myrealm',
  defaultMethod: 'token',
  methods: {
    oauth: { ... },
    token: { ... },
  }
}
```
Just my 2 cents, you can leave it as it is.

Reply:
isn't that what's already in there?
I don't see the difference

Reply:
Yea I guess I was just being pedantic here, it's already fine 🤙